frameworks/cassandra/tests/test_auth.py

```diff
@@ -22,12 +30,19 @@ def configure_package(configure_security: None) -> Iterator[None]:
             sdk_jobs.install_job(job)
 
         create_secret(
-            secret_value=config.SECRET_VALUE, secret_path=config.PACKAGE_NAME + '/' + config.SECRET_VALUE
+            secret_value=config.SECRET_VALUE,
+            secret_path=config.PACKAGE_NAME + "/" + config.SECRET_VALUE,
         )
         service_options = {
             "service": {
                 "name": config.SERVICE_NAME,
-                "security": {"authentication": {"enabled": True, "superuser": {"password_secret_path": "cassandra/password"}}, "authorization": {"enabled": True}},
+                "security": {
+                    "authentication": {
+                        "enabled": True,
+                        "superuser": {"password_secret_path": "cassandra/password"},
```

Comment:
looks like you're setting the same secret path in 2 different ways. Can you create a var `secret_path=config.PACKAGE_NAME + "/" + config.SECRET_VALUE` and use it both here and on L34?